coalib/bearlib/abstractions/ExternalBearWrap.py

```diff
@@ -0,0 +1,69 @@
+from functools import partial
+import json
+import inspect
+
+from coalib.bears.LocalBear import LocalBear
+from coala_decorators.decorators import enforce_signature
+from coalib.misc.Shell import run_shell_command
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+from coalib.settings.FunctionMetadata import FunctionMetadata
+
+
+def _create_wrapper(klass, options):
+    class ExternalBearWrapBase(LocalBear):
+
+        @staticmethod
+        def get_executable():
+            """
+            Returns the executable of this class.
+
+            :return:
+                The executable name.
+            """
+            return options["executable"]
```

Comment:
I prefered to keep it like that so it looks like linter. Maybe in the future be merged.